tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -137,6 +137,23 @@ exports[`capture_reassignable.w > stderr 1`] = `
 "
 `;
 
+exports[`classes_and_resources.w > stderr 1`] = `
+"error: Unknown symbol \\"init\\"
+  --> ../../../examples/tests/invalid/classes_and_resources.w:1:7
+  |
+1 | class C4 { x:num; } // class with field and no init
+  |       ^^ Unknown symbol \\"init\\"
```

Comment:
I'm not sure this error message makes sense to me - `init` is not written anywhere so it's not clear why the compiler is giving this error